DirectProgramming/DPC++FPGA/ReferenceDesigns/qrd/sample.json

```diff
@@ -8,6 +8,7 @@
   "builder": ["ide", "cmake"],
   "targetDevice": ["FPGA"],
   "languages": [{"cpp":{}}],
+  "targetDevice": [ "FPGA" ],
```

Comment:
This file already contains a target device (look at line 9).  Please remove this file from this PR.

Reply:
fixed